app/javascript/pages/account/my-applications.tsx

```diff
@@ -1,12 +1,139 @@
 import React from "react"
 import Layout from "../../layouts/Layout"
 import withAppSetup from "../../layouts/withAppSetup"
+import { t, Icon, Button } from "@bloom-housing/ui-components"
+import { Card, Heading } from "@bloom-housing/ui-seeds"
+import { ApplicationItem } from "../../components/ApplicationItem"
+import { getLocalizedPath } from "../../util/routeUtil"
+import { getCurrentLanguage } from "../../util/languageUtil"
 
 const MyApplications = () => {
+  const noApplications = () => {
+    return (
+      <Card.Section className="flex flex-col bg-primary-lighter items-center pb-12 border-t">
+        <h2 className="text-xl">{t("myApplications.noApplications")}</h2>
+        <div className="flex flex-col gap-y-4 w-3/5 pt-4">
+          <Button
```

Comment:
question: why not [LinkButton](https://storybook.bloom.exygy.dev/?path=/story/actions-link-button--standard)?